test/message-input.test.js

```diff
@@ -21,4 +25,68 @@ describe('message-input', () => {
   it('text area placeholder should be set', () => {
     expect(messageInput.shadowRoot.querySelector('vaadin-text-area').placeholder).to.be.equal('Message');
   });
+
+  describe('submit functionality', () => {
+    it('should fire a submit event when calling submitMessage()', () => {
+      const spy = sinon.spy();
+      messageInput.addEventListener('submit', spy);
+      messageInput.value = 'foo';
+      messageInput.__submit();
+      expect(spy.calledOnce).to.be.true;
+    });
```

Comment:
The function name in test description is out of date, but there's actually no reason to have the test case at all, now that the function is just a private implementation detail.
```suggestion
```